src/commands/utils/wallet.rs

```diff
@@ -0,0 +1,24 @@
+use zksync_ethers_rs::{
+    core::k256::ecdsa::SigningKey,
+    middleware::SignerMiddleware,
+    providers::{Http, Middleware, Provider},
+    signers::{LocalWallet, Signer, Wallet},
+    zk_wallet::ZKWallet,
+};
+
+pub(crate) async fn new_zkwallet(
```

Comment:
This function will be very useful. In the future, we can implement `From<ZKSyncConfig> for ZKWallet` and then just instantiate a `ZKWallet` with either of the following options
```rust
let zk_wallet = ZKWallet::from(cfg);
let zk_wallet: ZKWallet = cfg.into();
let zk_wallet = cfg.into::<ZKWallet>();
```

Reply:
working on it